src/ajitSJune20/FindDuplicateNumInIntAray.java

```diff
@@ -0,0 +1,31 @@
+//Find duplicate number on Integer array in Java?  Example : {0, 3,1, 2, 3}, then duplicated number is 3.
+package ajitSJune20;
+
+public class FindDuplicateNumInIntAray {
+
+	int getDuplicateNumInArray(int[] array) {
+		int dupNum = -1;
+		for (int index1 = 0; index1 < array.length; index1++) {
+			for (int index2 = (index1 + 1); index2 < array.length; index2++) {
+				if (array[index1] == array[index2]) {
+					dupNum = array[index2];
```

Comment:
We could have simplified it.. the moment if condition is true that means Duplicate no. is there so Print that time only.. and what if the duplicate no. is -1.. instead of -1 we can use boolean

Reply:
If there is no  duplicate number found then It will print "No duplicate element found in Array"
![image](https://user-images.githubusercontent.com/66831507/86025331-b201d700-ba4b-11ea-8eb4-df176bf9d838.png)

I agree we can use Boolean as well/